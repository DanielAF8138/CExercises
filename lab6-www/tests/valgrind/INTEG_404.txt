Received incoming request from 127.0.0.1
GET request for /non_existent.html
Bytes written to socket: -1
