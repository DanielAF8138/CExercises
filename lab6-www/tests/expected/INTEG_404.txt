Received incoming request from 127.0.0.1
GET request for /non_existent.html
Bytes written to socket: -1

++++++++++++++++++++++++++++
Client received
++++++++++++++++++++++++++++
HTTP/1.0 404 Not Found

++++++++++++++++++++++++++++
